ironfish/package.json

```diff
@@ -18,7 +18,7 @@
   ],
   "dependencies": {
     "@ironfish/bfilter": "0.0.1",
-    "@ironfish/rust-nodejs": "0.1.9",
+    "@ironfish/rust-nodejs": "0.1.10",
```

Comment:
Should the version bump for this be in a separate PR? Or should it bump the package.json of ironfish-rust-nodejs in this PR as well

Reply:
Oh, this is a fix that I merged into staging already that I duplicated here to keep testing: #1966